samples/net/updatehub/README.rst

```diff
@@ -0,0 +1,240 @@
+UpdateHub sample
+################
+
+Overview
+********
+
+UpdateHub is an enterprise-grade solution which makes it simple to remotely
```

Comment:
Could we have a link to specification of updatehub here?

There could be a link and chapter about updatehub in /doc/reference/networking/protocols.rst

Reply:
What do you mean by specification here?

Reply:
> What do you mean by specification here?

For example LWM2M documentation has links to relevant specifications, see example here https://builds.zephyrproject.org/zephyrproject-rtos/zephyr/14180/docs/reference/networking/lwm2m.html#overview
I was wondering that if someone is reading this sample documentation via browser, he/she could just follow link to some webpage that could explain things about updatehub. Even wikipage etc. would be useful.

Reply:
We have https://docs.updatehub.io/; this needs to have a Zephyr specific session added but seems to be the most logical place to point the user to.

Reply:
yes, that looks exactly what I meant